src/apiCalls.js

```diff
@@ -1,8 +1,26 @@
-fetch('https://api.ebird.org/v2/product/spplist/US-NV-001', {
-    headers: {
-      "X-eBirdApiToken": "f5n5l2qml9if"
-    }
-  })
-    .then(res => res.json())
-    .then(data => console.log(data))
-    .catch(err => console.error(err))
\ No newline at end of file
+const getBirdKeysByLocation = async (region) => {
+    const res = await fetch(`https://api.ebird.org/v2/product/spplist/${region}`, {
+        headers: {
+            "X-eBirdApiToken": "f5n5l2qml9if"
+        }
+    })
+    return await res.json()
+}
+
+const getBirdsData = (keys) => {
+    let birdsData = [];
+    keys.forEach(async key => {
+        let thisBirdData;
+        await getBirdData(key).then(result => thisBirdData = {...result[0], isChecked: false})
+        birdsData.push(thisBirdData)
+    });
```

Comment:
I read that async functions don't work with forEach loop, try using a for of loop instead, it worked on my end and you can then remove the setTimeout function.